tests/foreman/api/test_contentview.py

```diff
@@ -310,8 +341,9 @@ def test_positive_publish_3(self):
 
         # Publish the content view several times and check that each version
         # has the puppet module added above.
-        for _ in range(REPEAT):
+        for i in range(random.randint(3, 5)):
```

Comment:
I very much like random data. However, publishing and promoting content views takes an enormously long time. Here's the comment for `REPEAT` (it should be turned in to a docstring):

    # Some tests repeatedly publish content views or promote content view versions.
    # How many times should that be done? A higher number means a more interesting
    # but longer test.

Is there a compelling reason to use `random.randint` here?